guides/doc-Project_API/topics/proc_uploading-content-to-projectserver.adoc

```diff
@@ -0,0 +1,277 @@
+[id="uploading-content-to-{project-context}-server"]
+= Uploading Content to {ProjectServer}
+
+You can use the {Project} API to upload and import large files to your {ProjectServer}.
```

Comment:
does this work for any file type? Needs technical ACK pls.

Reply:
Yes, it does. Verified in APIdoc:
![image](https://github.com/user-attachments/assets/b398266b-0a61-46e3-a716-dd4041e78735)